sdk/resourcemanager/azure-resourcemanager-test/src/main/java/com/azure/resourcemanager/test/utils/TestIdentifierProvider.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.resourcemanager.test.utils;
+
+import com.azure.core.management.provider.IdentifierProvider;
+import com.azure.core.test.utils.TestResourceNamer;
+
+/**
+ * Class helps generate unique identifier.
+ */
+public class TestIdentifierProvider implements IdentifierProvider {
+
+    private final TestResourceNamer testResourceNamer;
+
+    /**
+     * Constructor of TestIdentifierProvider
+     *
+     * @param testResourceNamer the test resource namer
+     */
+    public TestIdentifierProvider(TestResourceNamer testResourceNamer) {
```

Comment:
Just wrap the TestResourceNamer into our TestIdentifierProvider as the ResourceNamer/TestResourceNamer are both set in azure-core-test.